### PR TITLE
Migrate frontend components to MUI.

### LINKED ISSUE #9796: Replace SUIR with MUI
Replace Semantic UI React with Material UI. 

POC: #9550.

Tasks:
- [x] Triangle subject overview title is too small.
- [x] Make border of dashboard cards grey, black on hover
- [x] Make blue darker (menu, footer, headers, buttons)
- [x] The center column and the logo in the footer are misaligned
- [x] Make sure that the headings in the settings panel don't line break
- [x] Don't indent the menu items in the settings panel
- [x] In the desired reaction times panel, make the help texts consistent (means versus colon).
- [x] In the issue tracker panel, make the Jira description wrap
- [x] Add a background color to the input fields to distinguish them more from the background
- [x] Change the anchor the the "Read the Docs" links to "Read the docs about the [item type]"
- [x] The warning icon when small needs a smaller triangle.
- [x] Use the same font in Victory charts as in the rest of the UI.
- [x] More padding between the source parameters (and other fields?)
- [x] Don't show selected items in "autocomplete" fields.
- [x] Change "Tags" to "Metric tags" in the metric configuration tab.
- [x] Use a lighter grey in the pie charts in dark mode.
- [x] Check metric column header alignment
- [x] Fix unit test warnings
- [x] Fix console warnings
- [x] Get rid of `STATUS_COLORS_RGB`.
- [x] Make subject type description wrap in the subject type picker in the subject title.
- [x] Check sorting of drop down menu's.
- [x] Make violation states description wrap in the measurement entity status dropdown.
- [x] Update screenshots.
- [x] Update changelog.

Out of scope:
- #10687
- #10688
- #10686
- #10690
- #10689

Won't do:

- Break long report/subject titles in dashboard cards: won't do, the pie charts will become very small if the report/subject title is long
- Make the delete button color red instead of orange: won't do because red is the color used for errors and orange for warnings; deleting something is not an error.

### PR DESCRIPTION
**Pull request by fniessink**
- StatusIcon.

Partially implements #9796.

### CODE REVIEW

File: components/frontend/src/metric/status.js [hunk 1]
```diff
@@ -23,6 +25,22 @@ export const STATUS_COLORS_RGB = {
...
+    debt_target_met: grey[500],
+    informative: blue[500],
+    unknown: grey[300],
+}
+export const STATUS_ICONS = {
+    target_met: <Check />,
+    near_target_met: <Warning />,
+    debt_target_met: <Money />,
+    target_not_met: <Bolt />,
+    informative: <b>i</b>,
```
Comment (wkoot) [inline components/frontend/src/metric/status.js:41]: There's no (i) icon?

File: components/frontend/src/metric/Target.js [hunk 2]
```diff
@@ -48,22 +48,20 @@ function debtTargetActive(metric, direction) {
...
-                <Header>
-                    <span>
-                        {STATUS_SHORT_NAME[status]} <StatusIcon status={status} size="tiny" />
-                    </span>
-                    <Header.Subheader>{capitalize(color)}</Header.Subheader>
-                </Header>
-                <b>{children}</b>
-            </Segment>
-        </Segment>
+        <Box sx={{ padding: "10px", border: `12px solid ${STATUS_COLORS_MUI[status]}` }}>
```
Comment (wkoot) [inline components/frontend/src/metric/Target.js:55]: Why inline styling here?

File: components/frontend/src/measurement/StatusIcon.js [hunk 3]
```diff
@@ -1,34 +1,28 @@
...
+    const sizes = { small: 20, undefined: 32 }
     const statusName = STATUS_SHORT_NAME[status]
-    const icon = <Icon aria-label={statusName} className={status} inverted circular name={icon_name} size={size} />
-    return status_start ? (
-        <Popup trigger={icon} flowing hoverable>
-            <TimeAgoWithDate date={status_start}>{`${statusName} since`}</TimeAgoWithDate>
-        </Popup>
-    ) : (
-        icon
+    // Use Avatar to create a round inverted icon:
```
Comment (wkoot) [inline components/frontend/src/measurement/StatusIcon.js:11]: What's a round inverted icon?

File: components/frontend/src/metric/status.js (hunk 1 above)
Comment (fniessink) [reply components/frontend/src/metric/status.js:41]: Surprisingly enough not. This works pretty well though.

File: components/frontend/src/metric/Target.js (hunk 2 above)
Comment (fniessink) [reply components/frontend/src/metric/Target.js:55]: It's the recommended way to override styling on a one-off basis, see https://mui.com/material-ui/customization/how-to-customize.

File: components/frontend/src/measurement/StatusIcon.js (hunk 3 above)
Comment (fniessink) [reply components/frontend/src/measurement/StatusIcon.js:11]: These are round inverted icons: image

**Pull request merged**